packages/workflow/src/records/handler/view.ts

```diff
@@ -26,12 +26,18 @@ export async function viewRecordHandler(
 
   const viewedRecord = await toViewed(record, token)
 
-  const viewedRecordWithTaskOnly: Bundle = {
+  const viewedRecordWithSpecificEntries: Bundle = {
     ...viewedRecord,
-    entry: [viewedRecord.entry.find((e) => e.resource.resourceType === 'Task')!]
+    entry: [
+      ...viewedRecord.entry.filter(
+        (e) =>
+          e.resource.resourceType === 'Task' ||
+          e.resource.resourceType === 'PractitionerRole'
+      )!
```

Comment:
Could you leave a comment here explaining why this is done, so that we remember the next time we land here?